src/azure-cli/azure/cli/command_modules/acs/_help.py

```diff
@@ -959,3 +959,30 @@
     text: |-
         az openshift wait -g MyResourceGroup -n MyManagedCluster --updated --interval 60 --timeout 1800
 """
+
+helps['openshift monitor'] = """
+type: group
+short-summary: Commands to manage Log Analytics monitoring. Requires "--workspace-id".
+"""
+
+helps['openshift monitor enable'] = """
```

Comment:
Is `openshift log enable` better? Monitor is a quite big scope. Will you support other monitor-related feature later? or it can be a command like `openshift enable-log` instead of `openshift monitor enable`.

Reply:
I don't think `log` is a better name. Log Analytics is only part of Azure Monitor and this is what this PR is intended for. 
This PR creates a subgroup `monitor`, so it is extensible and can in future support more features.

Reply:
It is also more in line with how AKS achieves the same thing